### ISSUE TITLE
Commsat model missing polys from lower solar panel array

### ISSUE THREAD

**Issue by thaddeus-ex** [Freelancer-Sirius-Revival/FLSR#959]
![telescope1](https://github.com/user-attachments/assets/320f162c-6588-4e82-859a-8dbd6f67824f)
![telescope2](https://github.com/user-attachments/assets/3a4b5f7a-fa41-4c2e-ba24-73f75e99ec12)

Missing polys on upper and lower faces of single lower panel. these are missing and not simple UV flipped, problem persists on upscale model commsatx4. Not sure I agree with modded smooth version. Step in the body denote revolving radar array with bearing surfaces culled for detail reduction - same as the big "X" weapons platform.

**Comment by thaddeus-ex** [comment]
![commsat03](https://github.com/user-attachments/assets/3b79703b-4ed8-441a-b28b-914bd3e8ca9c)

This thing is full of holes

**Comment by erikroe** [comment]
I have a downscaled version of it in my "modular" objects directory. If you'd fix that one it would be great there, too.

**Comment by thaddeus-ex** [comment]
Yes, planning a straight vanilla rebuild - mostly done, an animated .cmp version and the x4 version. with revised surs.
